pkg/blockstorage/azure/azuredisk.go

```diff
@@ -383,3 +452,69 @@ func (s *adStorage) SetTags(ctx context.Context, resource interface{}, tags map[
 		return errors.New(fmt.Sprintf("Unknown resource type %v", res))
 	}
 }
+
+func (s *adStorage) FromRegion(ctx context.Context, region string) ([]string, error) {
+	return staticRegionToZones(region)
+}
+
+func staticRegionToZones(region string) ([]string, error) {
```

Comment:
The https://godoc.org/github.com/Azure/azure-sdk-for-go/services/compute/mgmt/2019-07-01/compute#NewResourceSkusClient API appears to provide a programatic way of obtaining supported resource SKUs.  The only filter that appears to work from the command line is that for location, which would meet our needs. e.g.
`az rest -m get -u "/subscriptions/c29e991c-1987-4d14-bf47-ff05f49312fd/providers/Microsoft.Compute/skus?api-version=2019-07-01\&\$filter=location eq 'westus2'"`